index.js

```diff
@@ -219,6 +219,32 @@ class PSPDFKitView extends React.Component {
              NativeModules.PSPDFKitViewManager.addAnnotations(annotations, findNodeHandle(this.refs.pdfView));
         }
     }
+    
+    /**
+     * Gets the value of the form element of the fully qualified name.
+     * 
+     * @param fullyQualifiedName The fully qualified name of the form element.
+     *
+     * Returns a promise resolving a dictionary with the following structure:
+     * {'value' : value} or ['error' : Failed to get the form field value.]
```

Comment:
`['error' : Failed to get the form field value.]` is still shown as an array literal, while it should be an object.